pkgs/development/compilers/cudatoolkit/default.nix

```
@@ -56,5 +57,19 @@ in rec {
     gcc = gcc9;
   };
 
+  cudatoolkit_11_3 = common {
+    version = "11.3.1";
+    url = "https://developer.download.nvidia.com/compute/cuda/11.3.1/local_installers/cuda_11.3.1_465.19.01_linux.run";
+    sha256 = "0d19pwcqin76scbw1s5kgj8n0z1p4v1hyfldqmamilyfxycfm4xd";
+    gcc = gcc9;
+  };
+
+  cudatoolkit_11_4 = common {
+    version = "11.4.1";
+    url = "https://developer.download.nvidia.com/compute/cuda/11.4.1/local_installers/cuda_11.4.1_470.57.02_linux.run";
+    sha256 = "0180pb1zfajb9l6blr467xkx01yp3snfwm2xix8x52crf6d36v6x";
+    gcc = gcc11;
```

```suggestion
    gcc = gcc10; # can bump to 11 along with stdenv.cc
```
Ok let's stick with this for now. LGTM otherwise.